tools/workflow/clean_aws_rsc.go

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package main
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/elbv2"
+	"log"
+	"time"
+	"strings"
+)
+const (
+	containLbString = "aoc-lb"
+	pastDayDelete = 5
+)
+
+func main() {
+
+	log.Printf("Beging terminating EC2 Instances")
+	terminateEc2InstancesError := terminateEc2Instances()
+	if (terminateEc2InstancesError != nil){
+		log.Fatalf("Exit terminating EC2 Instances")
```

Comment:
This code was written before this pr. If a change is need, it should be in another pr not this one.

Reply:
The point of returning an error instead of just panicking in the function is to be able to give the user the option to handle the error themselves. If you're going to log the error and then return it, why not just use `log.Fatalf` right away?

Reply:
In terms of user's perspective, reasonable for me. Fixed it in the last commit @sethAmazon  @jefchien